src/docs/self-hosted/support.mdx

```diff
@@ -2,7 +2,7 @@
 title: 'Self-Hosted Support'
 ---
 
-We strive to provide a great self-hosted experience for folks willing to try Sentry that way. The [self-hosted repository](https://github.com/getsentry/self-hosted) is geared towards low to medium loads with simplicity in mind. We believe keeping this setup simple is important to show
+We strive to provide a great self-hosted experience for folks willing to try Sentry that way. The [self-hosted repository](https://github.com/getsentry/self-hosted) is geared towards low traffic loads (less than ~1 million submitted Sentry events per month). We believe keeping this setup simple is important to show
```

Comment:
Just curious, where did you get this number from?

Reply:
Observing my own instance as well as a few of my friends. 

I found that if you need to go beyond 1 million events, you'd tweak retention settings or bump the machine resource. 

It's very subjective, but I guess that's what the server can handle with the default machine resource.